Validator/Unique.php

```diff
@@ -0,0 +1,39 @@
+<?php
+namespace Vanio\DomainBundle\Validator;
+
+use Symfony\Component\Validator\Constraint;
+
+/**
+ * @Annotation
+ */
+class Unique extends Constraint
+{
+    const NOT_UNIQUE_ERROR = '';
+
+    protected static $errorNames = array(
+        self::NOT_UNIQUE_ERROR => 'NOT_UNIQUE_ERROR',
+    );
+
+    public $fields;
```

Comment:
A asi by bylo nejlepsi to fields povolit ve dvou formatech. klic jako jmeno property v $object resp. v tom commandu a hodnota jako jmeno property v ty entite. Precijen se to v tom commandu muze jmenovat i jinak, coz mi predtim nedoslo. Kdyz se neuvede klic (je numeric), tak by to mohlo znamenat, ze se to jmenuje stejne, at se to zbytecne nepise dvakrat.